### ISSUE TITLE
Cannot compile Framework

### ISSUE THREAD

**Issue by daviddelmonte** [audiokit/AudioKit#663]
It's probably me. However, in case...

I downloaded the latest AudioKit package.

Then I used Terminal.app to 

1. cd to Frameworks Folder.

2.  ./build_frameworks.sh

I got this terminal output:

```
Building frameworks for platforms: iOS macOS tvOS
Build settings from command line:
    BUILD_DIR = /Volumes/HD1 .... /3rdparty/AudioKit-master/Frameworks/build
    SDKROOT = iphonesimulator10.2

Build settings from configuration file 'simulator.xcconfig':
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_REQUIRED = NO
    ENABLE_BITCODE = NO
    IPHONEOS_DEPLOYMENT_TARGET = 9.0
    ONLY_ACTIVE_ARCH = NO

xcodebuild: error: '../AudioKit/iOS/AudioKit for iOS.xcodeproj' does not exist.
Davids-Mac-Pro-3:Frameworks david$ 
```

Please advise

**Comment by aure** [comment]
It seems like you have an incomplete copy of AudioKit, probably you need to download the source code and instead you downloaded the framework itself?  I've tested this and its working so I will close the issue (for others) but we can still correspond by adding to the issue.

**Comment by daviddelmonte** [comment]
Thanks Aure, that was it

**Comment by daviddelmonte** [comment]
Sorry to pester you Aure.. I’m getting these messages on App Store validation.. Did I miss something else? Thanks

oh- I’m using Xcode 8 GM


On Dec 18, 2016, at 12:27 AM, Aurelius Prochazka <notifications@github.com> wrote:

Closed #663 <https://github.com/audiokit/AudioKit/issues/663>.
